src/WebJobs.Script/Config/HostHealthMonitorConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class HostHealthMonitorConfiguration
+    {
+        public HostHealthMonitorConfiguration()
+        {
+            Enabled = true;
+
+            // these default numbers translate to a 50% health rating
+            // over the window.
+            HealthCheckInterval = TimeSpan.FromSeconds(15);
+            HealthCheckWindow = TimeSpan.FromMinutes(15);
+            HealthCheckThreshold = 30;
+        }
```

Comment:
I want to reiterate the comment I left on #2115 - to me it looks like this will take **much too long** to detect an unhealthy state. I would like something more like:
- interval = 10 seconds
- window = 2 minutes
- threshold = 6